index.xml

```diff
@@ -6368,6 +6390,17 @@ iframe.appendChild instanceof w.Function;  <span class='comment'>// Evaluates to
               for Web pages.
             </p>
           </div>
+
+          <p>
+            Although at the time of this writing the ECMAScript specification does not reflect this,
+            every ECMAScript object must have an <dfn data-export=''
+            id='dfn-associated-realm'>associated <a class='external'>Realm</a></dfn>. The mechanisms
+            for associating objects with Realms are, for now, underspecified. However, we note that
+            in the case of <a class='dfnref' href='#dfn-platform-object'>platform objects</a>, the
+            associated Realm is equal to the object's <a class='external'>relevant Realm</a>, and
+            for function objects the associated Realm is equal to the Realm returned by <a
+            class='external'>GetFunctionRealm</a>.
```

Comment:
Well.  For the honest-to-God Function case there is an obvious answer that everyone agrees on and we should spec that.

For the other cases, we should try to see what various engines do in practice and see if we can get feedback from other implementors as to what they think should happen.